### ISSUE TITLE
Typegen fails on 8.0.1

### ISSUE THREAD

**Issue by nathanwhit** [polkadot-js/api#4737]
After updating a project to the latest api version I ran into trouble trying to generate typescript types. Running

```bash
ts-node --skip-project node_modules/.bin/polkadot-types-from-defs --package mypackage/interfaces --input ./src/interfaces --endpoint ./mymetadata.json
```
now fails with `Error: Cannot find module '../src/fromDefs.ts'`.

I dug into the error a bit and found that we're hitting [this catch statement](https://github.com/polkadot-js/api/blob/66159955ac427ec4dfd2ffdb611dae7159c57663/packages/typegen/scripts/execWithSrc.cjs#L9) with the error: 

```
Error: ENOENT: no such file or directory, open 'mypackage/node_modules/@polkadot/typegen/cjs/templates/header.hbs'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at readTemplate (mypackage/node_modules/@polkadot/typegen/cjs/util/file.js:35:22)
    at Object.<anonymous> (mypackage/node_modules/@polkadot/typegen/cjs/util/formatting.js:81:62)
    at Module._compile (node:internal/modules/cjs/loader:1099:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
```
(where `mypackage` is the path to my project, just replaced for privacy)

**Comment by jacogr** [comment]
Umpf. Yes, it is broken. Will need a patch release.

**Comment by jacogr** [comment]
Fix here https://github.com/polkadot-js/api/pull/4738/files#diff-1e1848f1e213ca5973e744d62dc27b7804b8c879f44ddee82b25d7a9d2613305R26 - will be released as 8.0.2 from CI when merged.
